viewport-segments-api/index.css

```diff
@@ -116,7 +116,7 @@ h1 {
 
 /* Media queries */
 
-@media (horizontal-viewport-segments: 1) and (orientation: landscape) {
+@media (horizontal-viewport-segments: 1) and (vertical-viewport-segments: 1) and (orientation: landscape) {
```

Comment:
Just remove the MQ matching horizontal/vertical-viewport-segments: 1 it's pointless.

Reply:
Done.